lib/Documentable/CLI.pm6

```diff
@@ -120,10 +121,14 @@ package Documentable::CLI {
         Bool :a(:$all)             = False                    #= Equivalent to -t -p -s -i --search-index
     ) {
         my $beginning = now; # to measure total time
-        if (!"./html".IO.e || !"./assets".IO.e || !"./templates".IO.e and $v) {
-            note q:to/END/;
-                (warning) html and/or assets and/or templates directories
-                cannot be found. You can get the defaults by executing:
+	my $asset-dir-missing = @default-asset-dirs.map({!.IO.e}).any;
```

Comment:
weird indentation here?

Reply:
That's odd.  I usually avoid tabs like the plague.  Looks like something weird went on there.  I'll update and force-push.

Reply:
Ok, that's really odd. I usually don't use tabs at all, however tabs were definitely in the original patch.  Sorry about that!  I've replaced the tabs with spaces, but the indenting is still weird-looking on GitHub (it's fine in my vim).  No idea what's going here...

Reply:
Found the problem: hadn't pushed the right set of changes. Duh.